mesheryctl/internal/cli/root/app/import.go

```diff
@@ -0,0 +1,297 @@
+package app
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/asaskevich/govalidator"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/layer5io/meshery/models"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var importCmd = &cobra.Command{
+	Use:   "import",
+	Short: "Import app manifests",
+	Long:  `Import the app manifest into Meshery`,
+	Args:  cobra.MinimumNArgs(0),
+	Example: `
+// Import app manifest 
+mesheryctl app import -f [file/URL] -s [source-type]
+	`,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Should we have a common util function for getting the source types as well? Because this exact block in also present in the `onboard` command.

Reply:
Good idea

Reply:
Done